middlewares/index.js

```diff
@@ -77,5 +77,22 @@ module.exports = {
         message: 'Not Authorized'
       });
     }
+  },
+
+  requireParameters: function(paramList) {
+    return function(req, res, next) {
+      const { body, query } = req;
+
+      for(let i = 0; i < paramList.length; i++) {
+        const param = paramList[i];
+        if(!body.hasOwnProperty(param) && !query.hasOwnProperty(param)) {
```

Comment:
missing space after `if`
`if (!body.hasOwnProperty(param) || !query.hasOwnProperty(param))` ||instead of &?

Reply:
this should again go to eslint :)

Reply:
If we can find the param in either query params or body, it's fine. But it should not be missing from both.